### PR TITLE
Js Assignment 7 and 9

### PR DESCRIPTION
**Pull request by priyaughade**

### CODE REVIEW

File: priya/javascript/Assignment7/JsAssign7Q2.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Bubble_sort</title>
```
Comment (poonam-khode) [inline priya/javascript/Assignment7/JsAssign7Q2.html]: change title

File: priya/javascript/Assignment7/JsAssign7Q2.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  </head>
+  <body>
+    <p style="text-align: center">
+      Number_of_student :<input type="text" id="num" placeholder="Enter number of students"><br>
+      <br>
+      <button onclick="bubble()">submit</button><br>
+      <br>
+    </p>
+    <script>
+      function bubble() {
```
Comment (poonam-khode) [inline priya/javascript/Assignment7/JsAssign7Q2.html]: change function name and remove extra spaces in this file

File: priya/javascript/Assignment7/JsAssign7Q4.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        <p style="text-align: center;">
+            <h4>Selection_sort</h4>
+            Number_of_student : <input type="text" id="num" placeholder="Enter number of students"><br>
+            <br>
+            <button onclick="Select()">submit</button><br>
+            <br>
+            <p id="demo"></p>
+        </p>
+        <script>
+            function Select(){
```
Comment (MKadeer17) [inline priya/javascript/Assignment7/JsAssign7Q4.html]: Function name in lowercase.

**Review by MKadeer17 — APPROVED** [review]
Only variable name like: change "i" to some name like index or something else.

File: priya/javascript/Assignment7/JsAssign7Q3.html [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+                document.write("<br>***************************************" );
+                for (var i=0; i< arr.length-1; i++){
+                    for(var j=0; j<= arr.length-1; j++){
+                        if (arr[j] > arr[j+1]){
+                            temp = arr[j];
+                            arr[j]=arr[j+1];
+                            arr[j+1]=temp;
+                        }
+                    }
+                    document.write( "<br>" )
```
Comment (poonam-khode) [inline priya/javascript/Assignment7/JsAssign7Q3.html]: no need to write extra lines for break tag you can add it in single document.write

**Review by poonam-khode — CHANGES_REQUESTED** [review]
give proper space before and after operators in all files

File: priya/javascript/Assignment7/JsAssign7Q4.html (hunk 3 above)
Comment (priyaughade) [reply priya/javascript/Assignment7/JsAssign7Q4.html]: changes done

File: priya/javascript/Assignment7/JsAssign7Q2.html (hunk 1 above)
Comment (priyaughade) [reply priya/javascript/Assignment7/JsAssign7Q2.html]: changes done

File: priya/javascript/Assignment7/JsAssign7Q2.html (hunk 2 above)
Comment (priyaughade) [reply priya/javascript/Assignment7/JsAssign7Q2.html]: changes done

File: priya/javascript/Assignment7/JsAssign7Q3.html (hunk 4 above)
Comment (priyaughade) [reply priya/javascript/Assignment7/JsAssign7Q3.html]: changes done

**Review by poonam-khode — APPROVED** [review]

**Review by Snehal-113 — APPROVED** [review]

**Pull request merged**